Library/Formula/open-mesh.rb

```diff
@@ -16,11 +16,17 @@ class OpenMesh < Formula
   head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt" => :optional
```

Comment:
Yes, essentially.

If you do `brew info` on something that has an optional dep you'll see there's an automatic `:optional` option. [Caudec](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/caudec.rb) has about 16 optional deps but no `option` lines, and unless there's an `option` line given to override the default we print:

```
==> Options
--with-aacgain
	Build with aacgain support
--with-alac
	Build with alac support
--with-cksfv
	Build with cksfv support
--with-coreutils
	Build with coreutils support
--with-eye-d3
	Build with eye-d3 support
--with-ffmpeg
	Build with ffmpeg support
--with-flac
	Build with flac support
--with-flake
	Build with flake support
```

And so on inside `brew info caudec`.